### PR TITLE
Don't fetch chat twice

### LINKED ISSUE #1260: Bug: chatstore is not correctly cleared when account is switched?
somehow it tries to open a chat with the id of the chat that was open in the last account.

### PR DESCRIPTION
**Pull request by nicodh**
resolves #1260

### CODE REVIEW

**Comment by nicodh** [comment]
Interesting - so we have completely different behaviour.
I will investigate further...

File: src/renderer/components/message/MessageList.js [hunk 1]
```diff
@@ -47,7 +47,6 @@ export default function MessageList ({ chat, refComposer, locationStreamingEnabl
   }, [scrollToLastPage, scrollHeight])
 
   useEffect(() => {
-    chatStoreDispatch({ type: 'SELECT_CHAT', payload: chat.id })
     isFetching.current = false
```
Comment (Jikstra) [inline src/renderer/components/message/MessageList.js]: Do we need the isFetching.current code?

**Comment by Jikstra** [comment]
@Simon-Laux can't reproduce your issue, works as expected for me. Besides this, selecting a chat is now also fast again :)

**Comment by nicodh** [comment]
I think my second issue fixed the bug mentioned in this ticket...

**Comment by Simon-Laux** [comment]
:+1: can confirm `master` works now nicely again

**Pull request merged**